cli/src/test/java/org/apache/iotdb/tool/integration/ImportCsvTestIT.java

```diff
@@ -16,9 +16,10 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.iotdb.tool;
+package org.apache.iotdb.tool.integration;
```

Comment:
this IT should be moved to cross-test module.
It is because cli module has no dependency of iotdb-server (in my memory).